### PR TITLE
[Logs] Update Log fields to add CASB Findings dataset

### PR DESCRIPTION
**Pull request by soheiokamoto**
This updates Log fields to add CASB Findings dataset. New fields are added to other datasets, too.

It is synced from internal `logshare-api` repo's fields templates.

### CODE REVIEW

**Review by bharatnc — APPROVED** [review]

File: content/logs/reference/log-fields/account/casb_findings.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+The descriptions below detail the fields available for `casb_findings`.
+
+{{<table-wrap>}}
+
+| Field | Value | Type |
+| -- | -- | -- |
+| AssetDisplayName | Asset display name (for example, 'My File Name.docx'). | string |
+| AssetExternalID | Unique identifier for an asset of this type. Format will vary by policy vendor. | string |
+| AssetLink | URL to the asset. This may not be available for some policy vendors & asset types. | string |
```
Comment (ranbel) [inline content/logs/reference/log-fields/account/casb_findings.md]: Is it possible to replace the "&" symbol with "and"? (per our Style Guide)

Comment (soheiokamoto) [reply content/logs/reference/log-fields/account/casb_findings.md]: We can. Need to be fixed at the source templates.

Comment (ranbel) [reply content/logs/reference/log-fields/account/casb_findings.md]: Cool, everything else looks good from a PCX perspective.

Comment (soheiokamoto) [reply content/logs/reference/log-fields/account/casb_findings.md]: Updating in the source template. Updated here.

**Review by ranbel — APPROVED** [review]

**Pull request merged**